src/cocotb/regression.py

```diff
@@ -460,6 +460,7 @@ def _score_test(self, test: Test, outcome: Outcome) -> Tuple[bool, bool]:
             result_pass = False
 
             if _pdb_on_exception:
+                print(result)
```

Comment:
`traceback.print_exception` might provide more utility.

Reply:
It looks like the print the same exact thing, but I can change it

Reply:
Oh actually had the wrong worktree loaded, but I realize now it's printing twice because it's in the test summary. It probably wasn't printing for us because we modify the log level of cocotb's logger